### ISSUE TITLE
Add notice if no usuable metadata was found

### ISSUE THREAD

**Issue by BjornW** [commonsmachinery/copyrdf-addon#8]
Not sure if this is something we need to do because it will interrupt the user flow and may be very annoying. So I'm adding this issue just as something to discuss and not something which would require immediate attention. 

When a user selects and image and clicks 'copy image with credits' the plugin attempts to find the metadata required for this. If no suitable metadata is found the user is currently not notified. 

I'd suggest we may prompt the user with a message with something like: "Sorry we could not find the credits to this image automagically, we've only copied the image. Feel free to notify us so we may fix this in the future". 
Add two buttons, one "Please fix this image" to send the url of the selected image to us and one "Don't bother me with this again!" which disables the prompting (could be a user preference toggled on/off in user preferences?).
This implies some sort of receiving end (Catalog). The data we receive can help us prioritize which sites we should support first or engage with.

@jonasob, @petli @artfwo  Feedback on this issue is very much appreciated

**Comment by artfwo** [comment]
The menu item only appears after the metadata has been collected or fetched from a remote oEmbed resource. If metadata _can_ be fetched, but that wasn't done yet (we begin loading oEmbed after page loads completely), the menu item won't appear either.

**Comment by BjornW** [comment]
Ok, so technically a user should always get the metadata needed if the menu-item is available? What do you think about the gathering more data part of my suggestion? Is there another way of doing this less obtrusive if people are willing to help us make the tools work better?

**Comment by artfwo** [comment]
Yes, the user will always get metadata if the item is available. Perhaps we can show a disabled menu item if there is no metadata to indicate that the action is currently not possible.

**Comment by BjornW** [comment]
I'd would prefer _not_ to show a disabled item. Somehow I keep on thinking how we may get some feedback from users of sites they would like to use, but are unable to do so for the moment due to incorrect markup because that might give us some valuable feedback.

**Comment by jonasob** [comment]
I like the idea of, of no valid metadata can be found, tell the user this
and give the option to easily enter author and licence. This would then
enable the pasting to work still, even with some manual input, and we could
include a check box "report this to CM so that this site can be supported
in the future".

Optionally with entering an email address to subscribe to a github issue
such that the user will be automatically notified when it works with that
site.

**Comment by artfwo** [comment]
That, or the user could enter missing metadata on paste.

**Comment by jonasob** [comment]
Yep, but on paste you've lost the window where you have that information
readily available :)
